File: frontend/lib/enums/risk.ts
export enum Risk {
  Low = 1,
  Medium,
  High,
  Critical,
  Unknown,
}

export const riskLabels: Record<Risk, string> = {
  [Risk.Low]: 'Low',
  [Risk.Medium]: 'Medium',
  [Risk.High]: 'High',
  [Risk.Critical]: 'Critical',
  [Risk.Unknown]: 'Unknown',
};

File: VocareWebAPI/UserManagement/Models/Enums/Risk.cs
namespace VocareWebAPI.UserManagement.Models.Enums
{
    public enum Risk
    {
        Low = 1, // Niski
        Medium = 2, // Średni
        High = 3, // Wysoki
        Critical = 4, // Krytyczny
        Unknown = 5, // This value is used when the risk level is not specified or cannot be determined
    }
}
